packages/invoices/src/OutboundShipment/DetailView/OutboundServiceLineEdit/OutboundServiceLineEdit.tsx

```diff
@@ -14,7 +14,7 @@ export const OutboundServiceLineEdit = ({
 
   return (
     <Modal title="Service Charges">
-      <span>:)</span>
+      <ServiceItemSearchInput width={300} onChange={() => {}} />
```

Comment:
Just added this for now to have a looksie - need to setup a nice msw system with storybook 🤔